src/components/common/YourCompany/BusinessContactInfo.vue

```diff
@@ -65,7 +65,8 @@ export default class BusinessContactInfo extends Mixins(CommonMixin) {
     if (isEqual(contactInfo, this.getBusinessContact)) return
 
     try {
-      if (this.isAlterationFiling || this.isFirmChangeFiling || this.isFirmConversionFiling) {
+      if (this.isAlterationFiling || this.isFirmChangeFiling || this.isFirmConversionFiling ||
```

Comment:
Changes for 12896

Reply:
When an if flows to the next line, the existing code style is this:
```
if (
  ...
) {
```

Reply:
Thanks!